data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,486 @@
+# Copyright (c) 2020 Nomadic Volcano
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+########################################################################
+
+#Investigate a theorized smuggling route and rescue Hai. You have to do at least three of these missions.
+
+#NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.
+
+mission "Hai Rescue: Yeertle Hint"
+	landing
+	name "Investigate a Were-Squirrel"
+	description "A truly outlandish rumor has been floating around that there might be a Hai hiding out on <destination>."
+	source
+		near "Betelgeuse" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+		not planet "Prime"
+	destination Prime
+	to offer
+		"combat rating" > 50
+		has "Hai Leaks Response 6: offered"
+		has "event: slightly lost"
+		not "Hai Rescue: Yeertle Family: offered"
+		not "event: hai-human treaty signed"
+		random < 50
+	on offer
+		conversation
+			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. Often this contains useful tidbits, such as which merchants are ticked off at the moment and whether you should expect to wait a while for refueling. Today, though, you overhear a captain telling an acquaintance a remarkable tale.`
+			`	Apparently, there is a man-beast on <planet>, and when the other captain laughs at the idea of a modern "werewolf", the first one is quick to clarify that it's more like a meter and a half tall "were-squirrel". There seems to be some confused discussion about the mythology of such beasts being revealed when caught staring at bright lights until a third captain, listening in the same as you, butts in to inform them that one of them stole that from a popular show a few years back.`
+			`	You stop listening on your final approach as the conversation descends into a good-natured argument. Nevertheless, it sounds like a lead on a possible stranded Hai that would be worth following up.`
+				accept
+
+mission "Hai Rescue: Yeertle Family"
+	landing
+	name "Return the Yeertle family"
+	description "Transport the Yeertle family to <destination>. Payment is <payment>."
+	source Prime
+	destination Icelake
+	passengers 5
+	repeat 3
+	to offer
+		"combat rating" > 50
+		has "First Contact: Hai: offered"
+		not "event: hai-human treaty signed"
+		"Hai Rescue: Yeertle Family (offer count)" < 3
+		not "Hai Rescue: Yeertle Family: offered"
```

Comment:
`repeat 3` and this offer count are redundant with one another. Given what the purpose seems to be, these missions shouldn't have `repeat 3` on them, and should then remove the check for if the mission has offered (as that is redundant with how the game already checks if a mission should offer).